### ISSUE TITLE
[Deviantart] How can I download as of the last page

### ISSUE THREAD

**Issue by joselpg** [4pr0n/ripme#585]
The issue is this:
I donwloaded a centain number de pictures, 9000 aprox. And then appeared a 404 error. This took me several hours. And I don´t want to start again from the beginning. There is one way to continue from the former point?
I tried using this nomeclature: http://xxxxx.deviantart.com/favourites/?offset=48 (example) insted of the normal link: http://xxxxx.deviantart.com/favourites. But starts from the beginning.

In the past I used DeviantART Favorites Downloader, and had this usefull option.
I think this would be a great new tool.

Do you know other solutions?

* Ripme v1.4.20

**Comment by rautamiekka** [comment]
Just keep 'Overwrite existing files' disabled, use 2 threads and a minimum of 12000 milliseconds timeout and you'll pick up from you left off sooner than it took you to download them in the first place. 9000 items is, to put it lightly, a lot, since it's 375 pages at 24 items a page.

One of the devs might be able to tell something better, if you wanna wait for their word.

**Comment by metaprime** [comment]
Yes, re-rip should get you what you want, you'll just need to leave it a while. I'll keep this scenario in mind as I work on upgrading the re-rip functionality.

**Comment by joselpg** [comment]
@rautamiekka I going to use that configuration the next time. Thanks

@metaprime Yes, It would be an excellent idea. At least to add a option for choose a page as a starting point. Because in my case it took me several hours only to arrive to the last point
